virtualbox/verify-install.sh

```diff
@@ -0,0 +1,3 @@
+test -d $HOME/Applications/"Virtualbox.app"
+
+# brew cask list | grep --quiet -x virtualbox
```

Comment:
I think that one was yours so I left it in because I didn't understand it, but it is deleted now :)